clients/admin-ui/src/features/data-discovery-and-detection/SearchInput.tsx

```diff
@@ -36,7 +36,9 @@ export const SearchInput = ({ value, onChange }: SearchInputProps) => {
         onChange={handleOnChange}
         className="w-full"
       />
-      <Button onClick={onClear}>Clear</Button>
+      <Button onClick={onClear} className="bg-[#f5f5f5] hover:!bg-[#d9d9d9]">
+        Clear
+      </Button>
```

Comment:
Is this going to be common for search inputs? Can this component be reused outside of d&d? Maybe move this to a more common directory for sharing across the board if that's the case.